pkg/util/utils.go

```diff
@@ -69,3 +70,23 @@ func CommaStringToStringMap(s string) (map[string]string, error) {
 
 	return kvMap, nil
 }
+
+// EndPointHavePort will check for the presence of the port in the given endpoint
+// It will return true and nil, if the endpoint is valid and port is present
+// It will return false and nil, if the endpoint is valid and port is missing
+// It will return false and error, if the endpoint itself is not valid.
+func EndpointHavePort(endpoint string) (bool, error) {
+	_, _, err := net.SplitHostPort(endpoint)
+	if err != nil {
+		if strings.Contains(err.Error(), "missing port") == true {
```

Comment:
I'm concerned about this because if they change the error string in the golang library, then this would not work anymore.

Instead before calling line 79, check if the endpoint has a `:` , and if they do not, then it _must_ be only the IP or host. If they do, then use the `net.SplitHostPort` call.

Reply:
@lpabon Now I have modified the API name to validateEndpoint and added following functionalities.

1) Validate the hostname and IP address
2) Add the default port number 9020, if port is missing. 

Now I am not looking for hardcoding error message from the go library api. Please review the updated patch.

Added the UT for the new APIs. Also attached the manual test result with px context create command in the PR description.